Dockerfile

```diff
@@ -1,27 +1,28 @@
-FROM ubuntu:15.10
+FROM ubuntu:14.04
```

Comment:
Got it. I guess it's just b/c 14.04 is the LTS release. I don't know enough about what v15 offers to ignore the stability, so makes sense to me!